CONTRIBUTING.md

```diff
@@ -34,6 +34,13 @@ Every PR/commit on this repository will be run through a build pipeline to ensur
 ### Your first contribution
 Unsure where to begin contributing to GHCL? You can start by looking through these
 
+### Adding any new package
+If you are adding any external package on project, please make you use `Pipenv’s Dependency Resolution Approach`
```

Comment:
Is there any documentation we can link to on how to use Pipenv for first time users?